boards/arm/nucleo_l073rz/nucleo_l073rz.dts

```diff
@@ -126,3 +126,7 @@
 		pinctrl-names = "default";
 	};
 };
+
+&rng {
```

Comment:
Maybe add `zephyr,entropy` choice as well.

Reply:
yes, it's done